yarn.lock

```diff
@@ -10050,7 +10050,7 @@ ajv@^6.1.0, ajv@^6.10.0, ajv@^6.10.2, ajv@^6.11.0, ajv@^6.12.2, ajv@^6.12.4, ajv
     json-schema-traverse "^0.4.1"
     uri-js "^4.2.2"
 
-ajv@^8.0.0, ajv@^8.0.1, ajv@^8.11.0, ajv@^8.8.0:
+ajv@^8.0.0, ajv@^8.0.1, ajv@^8.8.0:
```

Comment:
Can we squash these two versions?

Reply:
Could you elaborate, not sure what you mean. The `8.11` was replaced by `8.12`.

Reply:
Do you mean remove all these `ajv@^8.0.0, ajv@^8.0.1, ajv@^8.8.0`? Is there a `yarn` command that does it, or just by hand?

Reply:
I did manually by hand, in next commit [a6aee62](https://github.com/elastic/kibana/pull/155234/commits/a6aee62831bc344735682fc91dd2842e2373006d), hopefully it works.

If there is a `yarn` command for that, curious to learn.

Reply:
yarn will fix it if we re-generate the lockfile but it would also updates all dependencies, using semver constraints which will increase scope.  My understanding is this isn't an issue on newer versions of yarn.

We've been making a best effort approach in the interim.